gempython/utils/db/settings.py

```diff
@@ -0,0 +1,17 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "gempython.utils.db.settings.local")
+SECRET_KEY = 'ef7$_)r0$#(x)ovd^m_6_bevsonkaxalvrhds9@ilb_p0o&-xn'
+
+INSTALLED_APPS = [
+    'gempython.utils.ldqm_db',
+    ]
+DATABASES = {
+    'default': {
+        'ENGINE':  'django.db.backends.mysql',
+        'NAME':     'ldqm_tif_qc8_db',
+        'PORT':     3306,
+        'HOST':     'cosmicstandtif',
+        'USER':     'gemdaq',
+        'PASSWORD': 'gemdaq',
+    }
+}
```

Comment:
Your PR was created from your develop branch, and on that branch, both the [github](https://github.com/mexanick/cmsgemos/blob/develop/gempython/utils/db/settings.py) and [raw](https://raw.githubusercontent.com/mexanick/cmsgemos/develop/gempython/utils/db/settings.py) file have duplicate information
The file you referenced is the initial commit for the PR